src/passes/Heap2Local.cpp

```diff
@@ -1248,7 +1242,7 @@ struct Array2Struct : PostWalker<Array2Struct> {
       // type here unconditionally, since we know the allocation flows through
       // here, and anyhow we will be removing the reference during Struct2Local,
       // later.)
-      curr->type = nonNullStruct;
+      curr->type = Type(structType, NonNullable);
```

Comment:
Was this diff not part of another PR?

Reply:
Yes. That PR was merged, so now it appears here :(